src/control/server/mgmt_svc.go

```diff
@@ -213,3 +215,26 @@ func (svc *mgmtSvc) CreatePool(
 
 	return resp, nil
 }
+
+// DestroyPool implements the method defined for the Management Service.
+func (svc *mgmtSvc) DestroyPool(
```

Comment:
Do we have to perpetuate this odd formatting practice? It's not idiomatic Go and it's harder to read. I would argue that these functions should be taking structs rather than long parameter lists, but that's a different discussion.

Reply:
@liw has also raised this point, is the top example in https://www.reddit.com/r/golang/comments/7e8dl0/how_do_you_or_your_team_format_long_function/ better? if so then I will use that going forward.

Reply:
As a rule of thumb, I have found that if your function is taking so many parameters that you find the line scrolling sideways off the screen unless you add line breaks, it's a sign that your function is either doing too much or should maybe have more structured input. 

One of the comments on the reddit post you linked references the AWS SDK: e.g. https://github.com/aws/aws-sdk-go/blob/master/service/s3/api.go

It probably doesn't make sense for us to go quite to that extreme, but I think there are some good ideas in there for dealing with structured input to complex API calls.

Reply:
in this case we are not supplying a large number of parameters, it's just that the names are long (this seems to be a feature of using the protobuf conventions/generated code). we are using request/response structures for organising input and output already